README.md

```diff
@@ -7,11 +7,56 @@ Building:
 $ go get github.com/cockroachlabs/roachprod
 ```
 
-# Synopsis
+# Summary
 
-`roachprod` manages per-user clusters under the [cockroach-ephemeral](https://console.cloud.google.com/home/dashboard?project=cockroach-ephemeral) GCE project.
+* clusters are created under the [cockroach-ephemeral](https://console.cloud.google.com/home/dashboard?project=cockroach-ephemeral) GCE project.
+* anyone can connect to any port on VMs in `cockroach-ephemeral` **DO NOT STORE SENSITIVE DATA**.
+* cluster names are prefixed with the user creating them (eg: `roachprod create test` creates the `marc-test` cluster)
+* VMs have a default lifetime of 12h (changeable with the `-l` flag) and are deleted 6 hours after expiration.
+* default settings create 4 VMs (`-n 4`) with 4 CPUs, 15GB ram (`--machine-type=n1-standard-4`) using local SSDs (`--local-ssd`).
 
-Simple workflow:
+# Setting up a cluster using roachprod/roachperf
+
+```bash
+# Create cluster with 4 nodes and local SSD (last node is used as load generator by roachperf)
+$ roachprod create test -n 4 --local-ssd
+Creating cluster marc-test with 4 nodes
+OK.
+  marc-test-0001	marc-test-0001.us-east1-b.cockroach-ephemeral	10.142.0.8	35.196.94.196
+  marc-test-0002	marc-test-0002.us-east1-b.cockroach-ephemeral	10.142.0.5	35.196.95.207
+  marc-test-0003	marc-test-0003.us-east1-b.cockroach-ephemeral	10.142.0.7	35.196.151.250
+  marc-test-0004	marc-test-0004.us-east1-b.cockroach-ephemeral	10.142.0.6	35.196.194.230
+Syncing...
+
+# Add gcloud SSH key
+$ ssh-add ~/.ssh/google_compute_engine
+
+# Stage scripts and binaries using crl-prod
+$ crl-stage-binaries marc-test all scripts
+$ crl-stage-binaries marc-test all cockroach
+
+# Or using roachperf (eg: for your locally-built binary)
+$ roachperf marc-test put cockroach
+
+# Start cluster using roachperf.
+$ roachperf marc-test start
+marc-test: starting 3/3
+marc-test: initializing cluster settings 1/1
+SET CLUSTER SETTING
+
+# Check the admin UI
+# http://35.196.94.196:8080
```

Comment:
We might want to add a facility to do this. Something like `roachprod <cluster> admin-ui`.

Reply:
Yeah, at some point I'll start adding `crl-prod`/`roachperf` functionality in `roachprod`. Not quite yet though.